configs/CMakeLists.txt

```diff
@@ -1,16 +1,17 @@
 get_filename_component(CHAINER_COMPILER_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR} PATH)
 
 set(jsons
-  chxvm.json
-  chxvm_test.json
-  dldt.json
-  ngraph.json
-  nvrtc.json
-  tvm.json
+  chxvm
```

Comment:
I think I may put a new target before this element, so.. (please check the next comment)

Reply:
Good point. I'm feared about it too.